apps/desktop/src/renderer/components/Chat/components/SubagentInnerToolCall/SubagentInnerToolCall.tsx

```
@@ -0,0 +1,324 @@
+				exitCode={exitCode}
+				state={state}
+			/>
+		);
+	}
+
+	const { label, icon } = getToolMeta(normalized);
+	const description = getDescription(normalized, args);
+	const hasResult = result !== null && result.trim().length > 0;
+
+	// Read file: parse and display using the shared ReadFileTool component
+	const parsedReadFile = useMemo(
```

Biome (2.4.10)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)